packages/ncids-css/scss/components/nci-header/internal/_megamenu.scss

```diff
@@ -31,6 +31,7 @@
 	}
 
 	&__primary-pane {
+		padding-right: 40px;
```

Comment:
>Just adding grid-gap to the grid-row containing the 2 panes seems to solve the issue and align with the comps. This ticket is to update the examples, stories and documentation to add the gap, as well as some samples to determine the tests at various sizes.

Instead of doing this in CSS we want to utilize the class that already exists for this purpose.